### ISSUE TITLE
when I use rest api with jdk11 httpClient,no statuscode in response

### ISSUE THREAD

**Issue by newsmallyu** [confluentinc/ksql#5471]
when I use rest api with jdk11 httpClient,no statuscode in response
ksqldb version: 0.8.1

however ksql version 5.4.0  not,it's normal

```
public static void main(String[] args) throws IOException, InterruptedException {
        HttpClient httpClient = HttpClient.newBuilder().build();
        JSONObject jsonObject = new JSONObject();
        jsonObject.put("ksql","LIST STREAMS;");
        HttpRequest request = HttpRequest.newBuilder()
                .uri(URI.create("http://xxx.xx.xx.x/ksql"))
                .timeout(Duration.ofMinutes(1))
                .header(Constants.KeyStr.CONTENT_TYPE, Constants.KeyStr.KSQL_APPLICATION_JSON)
                .POST(HttpRequest.BodyPublishers.ofString(jsonObject.toJSONString()))
                .build();
        HttpResponse<String> httpResponse = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
        String body = httpResponse.body();
        System.out.println(httpResponse);
        System.out.println(body);
    }
```
Exception in thread "main" java.io.IOException: no statuscode in response

**Comment by big-andy-coates** [comment]
Hi @newsmallyu, thanks for reporting.

There have been some recent changes around the rest api over the last couple of releases.

Can you provide the output from the above program please?

Also, is there any chance you have the time to test on the 0.9.x release?  This issue may already have been resolved.

Thanks

**Comment by newsmallyu** [comment]
Hi @big-andy-coates , thanks you for your reply.
Yes,I tested version 0.9.0 just now, it's work for me.   Thank you very much !
Can you tell me the reason for the error in 0.8.1? 
```
Exception in thread "main" java.io.IOException: no statuscode in response
	at java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:565)
	at java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:119)
```

**Comment by big-andy-coates** [comment]
I can't for certain I'm afraid, I just knew there were changes in and around 0.8 and 09 releases that could cause and fix the issue. But glad it's fixed in 0.9.

Likely, 0.8 may well have worked if running on JDK 8
